docs/index.rst

```diff
@@ -6,20 +6,12 @@
 Flax documentation
 ==================
 
-Flax is a neural network library for JAX that is designed for flexibility and
+Flax is a neural network library and ecosystem for JAX that is designed for flexibility. Flax 
```

Comment:
Can you reformat this paragraph?
(`vscode` : install [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) and then `Alt-Q`)

Reply:
done (I used Emacs alt-q for this, hope you're OK with that)